src/api/app/services/rpmlint_log_extractor.rb

```diff
@@ -40,7 +40,13 @@ def call
   def retrieve_rpmlint_log_from_log
     @parse_internal_log_file = true
 
-    log_content = Backend::Api::BuildResults::Binaries.file(project, repository, architecture, package, '_log')
+    log_content = begin
+      Backend::Api::BuildResults::Binaries.file(project, repository, architecture, package, '_log')
+    rescue Backend::NotFoundError => e # do not crash if no `_log` file found, just return empty log_content
+      @parse_internal_log_file = false
+      '' if e.summary.include?("package '#{package}' has no logfile")
```

Comment:
```suggestion
      ''
```
Why do we have to check for this error message? Shouldn't we return `''` for every `Backend::NotFoundError`?

Reply:
We could do that as well: I wonder what if the endpoint would return different messages for different scenarios. In my case we catch only the error when it is exactly with that wording, in yours we catch them all. But I think it is okay if we handle them all for now. Applying..

Reply:
Hm... The error is catched always. Before, instead of returning `''` it would have returned `nil`...